README.md

```diff
@@ -34,7 +34,7 @@ To get started using EZBake, please add it to the `:plugins` key in your
 `project.clj`:
 
 ```clojure
-{:plugins [[puppetlabs/lein-ezbake "0.2.3"]]}
+{:plugins [[puppetlabs/lein-ezbake "1.2.1"]]}
```

Comment:
Could we use the Clojars Project! macro so that we don't always have to update this?

e.g.

```
[![Clojars Project](http://clojars.org/puppetlabs/trapperkeeper/latest-version.svg)](http://clojars.org/puppetlabs/trapperkeeper)
```